rest_api/tests/unit/test_submit_requests.py

```diff
@@ -172,7 +172,7 @@ class PostBatchTests(BaseApiTest):
             - a link property that ends in '/batches?id=a'
         """
         batches = Mocks.make_batches('a')
-        statuses = {'a': self.status.COMMITTED}
+        statuses = {'a': client_pb2.COMMITTED}
```

Comment:
No, the way Protobuf's referenced enums is a little confusing. You reference the wrapping element, not the name of the enum. This enum is at the root level of the proto file, so it is just `client_pb2.COMMITTED`.

(Confusingly, I am about to wrap this enum in a _message_ named `BatchStatus`, and then it _will be_ `client_pb2.BatchStatus.COMMITTED`, but that will be in a future PR).